examples/java/settings.gradle

```diff
@@ -0,0 +1,25 @@
+import org.gradle.internal.os.OperatingSystem
+
+pluginManagement {
+    repositories {
+        mavenLocal()
+        gradlePluginPortal()
+        String frcYear = '2019'
+        File frcHome
+        if (OperatingSystem.current().isWindows()) {
+            String publicFolder = System.getenv('PUBLIC')
+            if (publicFolder == null) {
+                publicFolder = "C:\\Users\\Public"
+            }
+            frcHome = new File(publicFolder, "frc${frcYear}")
+        } else {
+            def userFolder = System.getProperty("user.home")
+            frcHome = new File(userFolder, "frc${frcYear}")
+        }
+        def frcHomeMaven = new File(frcHome, 'maven')
+        maven {
+            name 'frcHome'
+            url frcHomeMaven
+        }
+    }
+}

```

Comment:
Why not just make a settings plugin that people should use?
Example usage:
https://github.com/GradleWeaver/better-build-file-names

I guess that leaves you in a catch 22 to download the settings plugin.

Reply:
Yup. So thats automatically a no go.